python-app/goss/action.yml

```diff
@@ -0,0 +1,57 @@
+name: Goss
+description: Test image with Goss
+
+inputs:
+  GOSS_IMAGE:
+    description: "name used for goss temporary image"
+    required: true
+  GOSS_FILES_PATH:
+    description: "path to goss file"
+    required: true
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: check for goss file
+      id: goss_file
+      run: |
+        if [[ -d ${{ inputs.GOSS_FILES_PATH }} ]]; then
+          echo goss_enabled=1 >> $GITHUB_ENV
+        else
+          echo goss_enabled=0 >> $GITHUB_ENV
```

Comment:
To be more consistent between actions, can you use `true` and `false` like in the docker action?